tailwind.config.js

```diff
@@ -95,8 +95,10 @@ module.exports = {
       20: '5rem',
       24: '6rem',
       32: '8rem',
+      36: '9rem',
```

Comment:
I'm curious, what were these additional sizes needed for?

Reply:
Mostly used them for padding/margins in some areas where the sizes seemed to match the spec better than jumping 2 rem